bundles/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/handler/ChaserThingHandler.java

```diff
@@ -206,8 +206,8 @@ public void dispose() {
     @Override
     public void updateSwitchState(ChannelUID channelUID, State state) {
         logger.trace("received {} for {}", state, channelUID);
-        if (channelUID.getId().equals(CHANNEL_SWITCH) && (state instanceof OnOffType)) {
-            this.isRunning = (OnOffType) state;
+        if (channelUID.getId().equals(CHANNEL_SWITCH) && (state instanceof OnOffType onOffCommand)) {
+            this.isRunning = onOffCommand;
```

Comment:
Sorry, I overlooked there that it is state:
```suggestion
        if (channelUID.getId().equals(CHANNEL_SWITCH) && (state instanceof OnOffType onOffState)) {
            this.isRunning = onOffState;
```